src/yetibot/api/aws.clj

```diff
@@ -0,0 +1,160 @@
+(ns yetibot.api.aws
+  (:require
+    [yetibot.core.schema :refer [non-empty-str]]
+    [schema.core :as sch]
+    [yetibot.core.config :refer [get-config]]
+    [cognitect.aws.client.api :as aws]
```

Comment:
I am not sure, I couldn't find any reference to it if it exists

Reply:
Cool, I was just curious if there were any existing specs we could leverage. Looks like there are a few at https://github.com/cognitect-labs/aws-api/search?q=spec&unscoped_q=spec but I'm not sure if they're relevant to us.

Reply:
Oh good catch!! wow I totally missed those lines when I was on the examples, I was too focused on the invocation operation I guess...
Well, let me see whether we can use these specs because it would totally make sense if they can ;-)!!